boards/arm/mps3/doc/mps3_an552.rst

```diff
@@ -0,0 +1,240 @@
+has been tested with version 11.12.57 (Nov  2 2020).
+
+Zephyr board options
+====================
+
+The MPS3 AN552 is an SoC with Cortex-M55 architecture. Zephyr provides support
+for building for both Secure and Non-Secure firmware.
+
+The BOARD options are summarized below:
+
++-------------------------------+-----------------------------------------------+
+|   BOARD                       | Description                                   |
++===============================+===============================================+
+| ``mps3/corstone300/an552``    | For building Secure (or Secure-only) firmware |
++-------------------------------+-----------------------------------------------+
+| ``mps3/corstone300/an552/ns`` | For building Non-Secure firmware              |
++-------------------------------+-----------------------------------------------+
+
+.. note::
+   Board qualifier must include the board name as mentioned above.
+   ``mps3/corstone300`` without the board name is not a valid qualifier.
+
+Hardware
+********
+
+ARM MPS3 AN552 provides the following hardware components:
+
+- ARM Cortex-M55
+- Soft Macro Model (SMM) implementation of SSE-300 subsystem
+- Memory
+
+  - 8MB BRAM
+  - 4GB DDR4 SODIMM (by default, upgradeable to 8GB)
+  - 16GB eMMC
+  - 8MB QSPI Flash
+
+- Debug
+
+  - P‐JTAG, F-JTAG, SWD, 4-bit trace, 16-bit trace
+  - Four serial ports over USB
+
+- Interface
+
+  - AHB GPIO
+  - UART
+  - SPI
+  - I2C
+  - I2S
+  - Color LCD serial interface
+  - Ethernet
+  - VGA
+
+- On-board Peripherals
+
+  - Color LCD
+  - 10 LEDs
+  - 8 Switches
+  - 2 user push buttons
+
+Supported Features
+===================
+
+The ``mps3/corstone300/an552`` board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+See the `MPS3 FPGA Website`_ for a complete list of MPS3 AN552 board hardware
+features.
+
+The default configuration can be found in
+:zephyr_file:`boards/arm/mps3/mps3_corstone300_an552_defconfig`.
+
+For more details refer to `MPS3 AN552 Technical Reference Manual (TRM)`_.
+
+Serial Port
+===========
+
+The MPS3 AN552 has six UARTs. The Zephyr console output by default, uses
```

Comment:
When reworking into a single document, instances like this can probably be worded as:

```suggestion
The MPS3 has six UARTs. The Zephyr console output by default, uses
```